attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -49,16 +43,6 @@ fn get_quote_ioctl(report_data: &Vec<u8>) -> Result<Vec<u8>> {
     }
 }
 
-// Return true if the TD environment can extend runtime measurement,
-// else false.
-fn runtime_measurement_extend_available() -> bool {
```

Comment:
I would keep this function and move the check from below into it even though it's only one line. We'll probably update this check later.

Reply:
ok. itt saved me one extra `cfg()` check with https://github.com/confidential-containers/guest-components/pull/572/commits/1c8291f20d801a1242c0cf7c549fa6a04a29b333 but I'm fine adding it back.